swan-lake/integration/get-started.md

```diff
@@ -150,23 +129,75 @@ Let's run this package in your terminal:
 ```
 
 Alternatively, you can use the built-in `Try it` feature by clicking on the `Try it` CodeLens above the service declaration on VS Code.
 
+You can implement advanced filtering by updating the service as demonstrated in the following example.
+
+```ballerina
+import ballerina/http;
+
+public type Country record {
+    string name;
+    string continent;
+    int population;
+    decimal gdp;
+    decimal area;
+};
+
+public type CountryResponse record {
+    string name;
+    string continent;
+    decimal gdpPerCapita;
+};
+
+final http:Client countriesClient = check new ("https://dev-tools.wso2.com/gs/helpers/v1.0/");
+
+service / on new http:Listener(8080) {
+
+    resource function get countries() returns CountryResponse[]|http:InternalServerError {
```

Comment:
in the previous sample we are returning error, but here we are changing it to `http:InternalServerError` without mentioning anything about it. 
Do we really need it? Can't we simply use error and avoid do on fail as well? @MaryamZi ?

Reply:
Here wanted to demonstrate Http status code handling for errors easily. However this sample has only a 500 case. Maybe we can add a comment explaining this and possibilities.

Reply:
For tutorials and most other learn content we switched to status code return types to reflect best practices. The Ballerina in action samples still have error returns though. I guess, since this is just the getting started sample, we could use error here.

But I agree, if we are sticking to status code returns, we have to at least have a sentence explaining it and maybe link to the BBE.

Reply:
Explained the error handling part as part of the content
https://github.com/ballerina-platform/ballerina-dev-website/pull/9885/files#diff-71ef1f1bd2372ac1b1aebd4c504ca9299b3f74cef31aff17e3ac97c60d066596R147